Octokit/Models/Request/SearchIssuesRequest.cs

```diff
@@ -337,4 +336,53 @@ public enum IssueTypeQualifier
         [Parameter(Value = "issue")]
         Issue
     }
+
+    public class RepositoryCollection : IEnumerable<string>
+    {
+        readonly ICollection<string> _repositories = new List<string>();
+
+        public IEnumerator<string> GetEnumerator()
+        {
+            return _repositories.GetEnumerator();
+        }
+
+        IEnumerator IEnumerable.GetEnumerator()
+        {
+            return GetEnumerator();
+        }
+
+        public void Add(string owner, string name)
```

Comment:
Fun fact - you can also have `Add(string nameWithOwner)` here and both sorts will happily work with the collection initializer syntax.

@haacked should we support both? I still like the succinctness of writing them inline rather than splitting them out...